canonical/ubuntu.com
# update link to tshirt ts&cs

## Done

- updated the Ts&Cs link on /kubernetes/kubernetes-operations-survey to point to an updated version of the Ts&Cs

## QA

- visit /kubernetes/kubernetes-operations-survey
- click the "Competition Terms & Conditions" link
- you should be take to [this PDF](https://pages.ubuntu.com/rs/066-EOV-335/images/Cloud%20Survey%202022%20Q4%20-%20Terms%20and%20Conditions%20-%20Canonical%2010.22%20.pdf)

## Issue / Card

Fixes https://github.com/canonical/ubuntu.com/issues/12310
